libs/ui/packages/native/src/components/Cards/InformativeCard/index.tsx

```diff
@@ -31,14 +52,15 @@ const CardContainer = (props: CardProps): React.ReactElement => {
         </Text>
       </Flex>
 
-      <Flex width={100} height={75}>
+      {imageUrl && (
```

Comment:
prefer a ternary `imageUrl ? (/*...*/) : null` here

Reply:
Isn't that syntax preferred for specific cases (eg. when evaluating an array length, or when there's need for fallback ?). In that case I can't see any scenario where this would be an issue.

Reply:
Alternatively to the ternary you can use `!!imageUrl && whatever` or `Boolean(imageUrl) && whatever`

Reply:
Basically when you do conditional rendering with a `&&` make sure that to the left of the `&&` you always have either `null` or a boolean.